Classes/Core/Functional/Framework/Frontend/ResponseContent.php

```diff
@@ -108,14 +103,6 @@ function (string $sectionIdentifier) {
         );
     }
 
-    /**
-     * @return array
-     */
-    public function getScope(): array
```

Comment:
Not sure about changes in the `Response` layer... probably currently not used, but might be useful in future™

Reply:
Adjust restrictions like
```
$queryBuilder->getRestrictions()->removeAll()->add(GeneralUtility::makeInstance(DeletedRestriction::class));```

Reply:
We can re-add later. You know... Git...